pyparlaclarin/refine.py

```diff
@@ -21,6 +21,16 @@ def _iter(root, ns="{http://www.tei-c.org/ns/1.0}"):
                 elif elem.tag == "u":
                     elem.tag = ns + "u"
                     yield "u", elem
+                elif elem.tag == "p":
```

Comment:
This if else hell could be refactored a bit, but maybe that's an issue for another day